src/main/java/tech/picnic/jolo/Loader.java

```diff
@@ -65,6 +66,11 @@ public void next(Record record) {
     return get().stream();
   }
 
+  /** Returns all objects loaded by this loader. */
```

Comment:
```suggestion
  /** Collects and then returns all objects loaded by this loader. */
```
The `Collector` may do anything, so I think that's worth calling out.